### PR TITLE
Bubble Sort

### PR DESCRIPTION
**Pull request by marijanbrvar**
Creating Bubble Sort algorithm in Ruby
- Algorithm in file bubble_sort.rb comparing pars of items in the array and swapping position if the condition is met.
- Algorithm in file bubble_sort_by.rb accepts the code block and applies the same sorting as the previous one!

### CODE REVIEW

File: bubble_sort.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        array[i], array[i + 1] = array[i + 1], array[i]
+        swapped = true
+      end
+    end
+    break unless swapped
+  end
+  array
+end
+
+p bubble_sort([4, 3, 78, 2, 0, 2])
```
Comment (rukundoeric) [inline bubble_sort.rb]: debugging codes should be removed before you submit

File: bubble_sort_by.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    break unless swapped
+  end
+  array
+end
+
+sorted_by_array = bubble_sort_by(%w[hi hello hey]) do |left, right|
+  left.length - right.length
+end
+
+p sorted_by_array
```
Comment (rukundoeric) [inline bubble_sort_by.rb]: debugging codes should be removed before you submit.

**Review by rukundoeric — CHANGES_REQUESTED** [review]
# Status: ♻️ CHANGES REQUIRED ♻️
_Great job guys so far 👏 💯 👍. However, there are few issues we need to address before we move on, please check the comments below._

- ### To Highlight
  - [x] The project is clean. 💯
  - [x] The project follows Git flow. 💯

- ### To Improve
  - [x] [MANDATORY] Kindly remove all debugging codes.
  - [x] [MANDATORY] In your README, provide instructions to be followed to set up and run this project locally. 
  - [x] [MANDATORY] Your License link is broken.

_Please make all the required changes before you submit another code review. If you have any problems or concerns, you can reach out to me on Slack: Rukundo Eric 🚀 . I wish you all the very best 🥇_

### _Keep working hard._
### _Happy Coding😃_
-----------------------------------------------------------------------------------------------------------------------------------------
**Regards,
[Rukundo Eric](https://rukundo-eric.herokuapp.com/) ,
[LinkedIn](https://www.linkedin.com/in/rukundo-eric/), 
Slack: Rukundo Eric**

**Review by acushlakoncept — APPROVED** [review]
# Status: PROJECT APPROVED  🎉 🚀

### Hi, @marijanbrvar and @tanzila-abedin   Congratulations on completing your project 👏 

I will go ahead and mark this as complete, feel free to merge when you are ready. All the best with the next project.

### Cheers and Happy coding! 💻 👏

(Feel free to contact me on slack(@acushla), if you have any additional question)
Uduak Essien
[Twitter](https://twitter.com/acushlakoncept) | [LinkedIn](https://www.linkedin.com/in/acushlakoncept/)

**Pull request merged**